package/MDAnalysis/auxiliary/base.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+
+class AuxReader(object):
+    """ Base class for auxiliary readers
+    Does iteration and aligning between trajectory ts/auxiliary step
+    Actual reading/parsing of data to be handled by individual readers.
+    """
+      
+    # TODO deal with changing/different units
```

Comment:
But it would be nice to later have some automated conversion like we have for the trajectory files.  We make some assumptions in those as well (for most people these are correct assumptions)